### PR TITLE
[LiveComponent] Add doc about the placeholder macro

### PR DESCRIPTION
**Pull request by smnandre**
Add documentation about  #1532 (placeholder macro)

### CODE REVIEW

File: src/LiveComponent/doc/index.rst [hunk 1]
```diff
@@ -2255,26 +2259,88 @@ real component once the page has loaded.
...
+
+    {# With the component tag #}
     {% component SomeHeavyComponent with { defer: true } %}
         {% block loadingContent %}Loading...{% endblock %}
     {% endcomponent %}
 
-To change the initial tag from a ``div`` to something else, use the ``loading-tag`` option:
+Both ``loading-template`` and ``loadingContent`` can be used in the template
+_calling_ the component. But if want to set this content only once, you can add a
+``placeholder`` macro directly in the component template.
```
Comment (weaverryan) [inline src/LiveComponent/doc/index.rst]: macro called placeholder

that makes it more clear that placeholder with name vs placeholder is an adjective

File: src/LiveComponent/doc/index.rst [hunk 2]
```diff
@@ -2255,26 +2259,88 @@ real component once the page has loaded.
...
+        {% for i in 1..props.size %}
+            <div class="loading-product">
+                ...
+            </div>
+        {% endfor %}
+    {% endmacro %}
+
+If you style the placeholder content (giving the rows a specific height),
+you can then avoid layout shifts when the component is fully loaded (and
+the loading content replaced by the real content).
```
Comment (weaverryan) [inline src/LiveComponent/doc/index.rst]: I think it’s a bit too long - the first example is probably enough. 

Also, I’m wondering if this should license the FIRST option of customizing loading content. I’d imagine that I would more often embed the loading content inside the template vs in the calling template.

so logic would be:

- to customize loading, you can add this macro
- Else, if you want to control loading when calling the component, use the block trick.
- And, btw, you can control the loading tag for edge cases, like when your element is a tr

File: src/LiveComponent/doc/index.rst (hunk 1 above)
Comment (smnandre) [reply src/LiveComponent/doc/index.rst]: "named" placeholder ?

File: src/LiveComponent/doc/index.rst (hunk 2 above)
Comment (smnandre) [reply src/LiveComponent/doc/index.rst]: I've reordered all .. tell me if better

**Comment by weaverryan** [comment]
Very nice - thank you Simon!

**Pull request merged**